static/cast/README.md

```diff
@@ -0,0 +1,30 @@
+## Cast file
+
+The file ended with `.cast` come from [Asciinema](https://asciinema.org/) library which records and display terminal sessions. 
```

Comment:
Here, maybe `files` should be used, we will have not only one cast file.

Reply:
Here is just to explan what is "cast file".

Reply:
come  -->  comes

display  -->  displays